src/components/ImportFromS3Dialog/get-annotation-from-aws.js

```diff
@@ -1,58 +1,48 @@
-import getSampleNameFromURL from "../../utils/get-sample-name-from-url"
 import isEmpty from "../../utils/isEmpty"
 import Amplify, { Storage } from "aws-amplify"
+import jsonHandler from "../../utils/file-handlers/recent-items-handler"
+function CheckIfAnnotationExist(result, folderToFetch) {
+  return result.find(
+    (element) => element.key === `${folderToFetch}/annotations/annotations.json`
+  )
+}
+
+function GetSampleFromAnnotation(json, samples) {
```

Comment:
`json` -> `fileJSON` or a destructuring e.g.

```javascript
function GetSampleFromAnnotation({ content: udtJSON }, samples) {
/* ... */
}
```

would make it easier to understand what "json" is